wiki/Tournaments/JOKE/4/en.md

```diff
@@ -0,0 +1,119 @@
+---
+tags:
+  - JOKE
+  - JOKE 4
+---
+
+# Jaye's One Key Event 4: The Great Escape
+
+![JOKE 4 banner](img/banner.jpg)
+
+**Jaye's One Key Event 4: The Great Escape** (***JOKE***) is a 1v1, double-elimination, global osu!mania 1-key tournament hosted by ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352). It is the fourth instalment in the JOKE series.
```

Comment:
the abbreviation is usually iteration-specific

```suggestion
**Jaye's One Key Event 4: The Great Escape** (***JOKE 4***) is a 1v1, double-elimination, global osu!mania 1-key tournament hosted by ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352). It is the fourth instalment in the JOKE series.
```

Reply:
Not entirely sure about this. We've kinda always referred to all of the JOKEs as just JOKE. Turned into a bit of an inside joke if you get me?